agent/tool-scripts/postprocess/pcp-postprocess

```diff
@@ -1,50 +1,80 @@
 #!/bin/bash
-dir=$1
-interval=$2
 
-if [ -z "$interval" ]; then
-	interval=3
+PROG=$(basename "${0}")
+tool_output_dir="${1}"
+if [[ -z "${tool_output_dir}" ]]; then
+    printf -- "%s: missing required tool output directory argument\n" "${PROG}" >&2
+    exit 1
+fi
+
+pushd ${tool_output_dir} > /dev/null
+if [[ ${?} -ne 0 ]]; then
+    printf -- "%s: tool output directory argument, '%s', is not a directory\n" "${PROG}" "${tool_output_dir}" >&2
+    exit 1
+fi
+
+if [[ ! -s "./interval" ]]; then
+    printf -- "%s: missing 'interval' file in tool output directory, '%s'\n" "${PROG}" "${tool_output_dir}" >&2
+    exit 1
+fi
+interval=$(cat "./interval")
+
+command -v pminfo > /dev/null
+if [[ ${?} -ne 0 ]]; then
+    printf -- "%s: missing required 'pminfo' command\n" "${PROG}" >&2
+    exit 1
+fi
+
+command -v pmdumptext > /dev/null
```

Comment:
@portante the use of pmdumptext here could be replaced with the more modern PCP utility pcp2csv(1) - its a python script, part of the pcp-system-tools package on RPM systems (and core pcp package on DEB boxen).  pmdumptext(1) is from the Qt-dependent, optional pcp-gui package, so you'll likely have fewer dependencies as well as many more formatting options.

Reply:
I'll be adding a separate commit to this patch set which will show only the changes to use `pcp2csv`.